src/Components/Player/Player.tsx

```diff
@@ -0,0 +1,61 @@
+export type Piece = {
+  character: string;
+  index: number;
+  size: Size;
+  player: Player;
+};
+
+export type Status = "P1" | "P2" | "not-in-play";
+
+export type Size = "small" | "medium" | "large";
+
+export type PlayerRecord = {
+  win: number;
+};
+export default class Player {
```

Comment:
No react component, file should be `.ts`

Reply:
fixed